src/avatar.js

```diff
@@ -237,7 +237,7 @@ function createAvatarComponent({ sources = [] }) {
                     style={imageStyle}
                     src={this.state.src}
                     alt={alt || name || value}
-                    title={title || name || value}
+                    title={title !== undefined ? title : name || value}
```

Comment:
not better `tittle ?? name ?? value` ?

Reply:
@krychaxp `??` would be the same as `title !== undefined && title !== null`, therefore you still wouldn't be able to pass `null` as the `title` to prevent a `title` value from being set.